### PR TITLE
GPU Check Fix#2

### PR DESCRIPTION
**Pull request by 5he1n**
Yet another

### CODE REVIEW

File: server_release_template/web_gui/js/app/nls/wizard.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ define({
         "driverSuccess": "Driver successfully registered",
         "firewallFailed": "Setting firewall rules failed",
         "firewallSuccess": "Firewall rules successfully set",
-        "YourGPUIs": "Your GPU is",
-        "GPUSupported": "Great! This GPU is probably supported!",
+        "YourGPUIs": "Your GPU:",
+        "GPUSupported": "\nGreat! This GPU is probably supported!", //Yes, '\n' must be in translation
         "GPUUnsupported": "We are sorry, but this card may be unsupported. You can try ALVR anyway and see if it works",
         "GPUUnknown": "Oops. Looks like your GPU is disconnected or you have potato instead",
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/nls/wizard.js:35]: You can remove the unused string

File: server_release_template/web_gui/js/app/setupWizard.js [hunk 2]
```diff
@@ -7,52 +7,30 @@ define([
...
-                case "Intel(R)":
-                    return '🔴 '+i18n.GPUUnsupported;
-                default:
-                    return '🟣 '+i18n.GPUUnknown;
+                if (/Intel/.test(gpu) || unsupportedGPURegex.test(gpu)) {
+                    return '🔴 '+ gpu +i18n.GPUUnsupported;
+                } else {
+                    return '🟢 '+ gpu +i18n.GPUSupported;
+                }
             }
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/setupWizard.js:33]: Some indentation issues.

**Review by zarik5 — CHANGES_REQUESTED** [review]
Some nits, but looks good. Waiting for testing on AMD cards.

File: server_release_template/web_gui/js/app/setupWizard.js [hunk 3]
```diff
@@ -7,52 +7,30 @@ define([
...
-                    if (unsupportedGPURegex.test(gpu.name)) {
-                        return '🔴 '+i18n.GPUUnsupported;
-                    } else {
-                        return '🟢 '+i18n.GPUSupported;
-                    }
-                case "Intel(R)":
-                    return '🔴 '+i18n.GPUUnsupported;
-                default:
-                    return '🟣 '+i18n.GPUUnknown;
+                if (/Intel/.test(gpu) || unsupportedGPURegex.test(gpu)) {
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/setupWizard.js:28]: You can move "Intel" check in the previous regex.

**Review by zarik5 — CHANGES_REQUESTED** [review]

File: server_release_template/web_gui/js/app/nls/wizard.js (hunk 1 above)
Comment (5he1n) [reply server_release_template/web_gui/js/app/nls/wizard.js:35]: Oh,yeah, I will

File: server_release_template/web_gui/js/app/setupWizard.js (hunk 3 above)
Comment (5he1n) [reply server_release_template/web_gui/js/app/setupWizard.js:28]: Sure, yeah

File: server_release_template/web_gui/js/app/setupWizard.js (hunk 2 above)
Comment (5he1n) [reply server_release_template/web_gui/js/app/setupWizard.js:33]: May be, was fixing on the go

**Review by zarik5 — APPROVED** [review]

**Comment by zarik5** [comment]
Keeping this PR sitting here is not that useful, I can merge and see what happens.

**Comment by igorrafael** [comment]
I can confirm that since this fix I get the correct message with my Vega 56: "Great! This GPU is probably supported!".

With version 12.5.0 I used to get the "GPU disconected" message.

**Comment by zarik5** [comment]
@igorrafael Thanks! I appreciate the feedback.

**Comment by 5he1n** [comment]
@igorrafael thanks a lot!

**Pull request merged**